### ISSUE TITLE
"print" is not defined [reportUndefinedVariable] (plus other built-in functions)

### ISSUE THREAD

**Issue by jordan-bravo** [microsoft/pyright#9172]
Pyright is reporting built-in Python functions such as `print` as not defined, listing the rule `reportUndefinedVariables`.  This is happening with other build-in functions such as:
- `len()`
- `bool()`
- `abs()`
- etc...

`example.py`
```python
print("hello")
```

![Screenshot from 2024-10-08 14-13-04](https://github.com/user-attachments/assets/8144095d-2e69-4206-8898-1ed47a4ee918)

Details:

Python version: `2.7.18`
Pyright version: `1.1.377`
Neovim version: `0.10.1`

Contents of `pyproject.toml`:
```toml
[tool.pyright]
venv = ".venv"
venvPath = "."
pythonVersion = "2.7"
```

**Comment by erictraut** [comment]
This is because you're using Python 2.7. Pyright works only with Python 3.x. See [this documentation](https://microsoft.github.io/pyright/#/features?id=limitations).
